addons/sale/models/sale.py

```diff
@@ -896,8 +902,8 @@ def write(self, values):
         ('invoiced', 'Fully Invoiced'),
         ('to invoice', 'To Invoice'),
         ('no', 'Nothing to Invoice')
-        ], string='Invoice Status', compute='_compute_invoice_status', store=True, readonly=True, default='no')
-    price_unit = fields.Float('Unit Price', required=True, digits=dp.get_precision('Product Price'), default=0.0)
+    ], string='Invoice Status', compute='_compute_invoice_status', store=True, readonly=True, default='no')
```

Comment:
probably revert this

Reply:
I don't see why it shouldn't be required.

By the way, maybe we could add a state 'not_applicable' for lines and sections to avoid having any pre/post-process on these lines.

Reply:
Isn't the suggested not_applicable state exactly the the line_type? This is what needs to be tested when we want to decide if we need to process a line a certain way.